packages/site/src/components/TokensView.js

```diff
@@ -0,0 +1,186 @@
+import { Route, Switch, useRouteMatch } from "react-router-dom";
+
+import React, { useState, useEffect } from "react";
+import { TokenForm } from "./TokenForm";
+import { InitialForm } from "./InitialForm";
+import {
+	getCoreColorsFromTokens,
+	getUpdatedTokensWithCoreColors,
+} from "../utils/coreColorTokens";
+
+export const TokensView = () => {
+	let match = useRouteMatch();
+
+	const appActions = {
+		create: "create",
+		update: "update",
+	};
+
+	const [selectedBrand, setSelectedBrand] = useState({
+		action: "",
+		brand: "",
+	});
+
+	const supportsshowSaveFilePicker = () =>
```

Comment:
Looks like this is only used below to set state for canSaveFile, so could we remove this line and on line 30 just do:
`const [canSaveFile] = useState(window.showSaveFilePicker);` because window.showSaveFilePicker when undefined is falsey.

Reply:
ohhh nice!!!

Reply:
Actually, it looks like useState calls the function so the file picker tries to run...there might be a way to get it working, but going to leave it as is for now.